test/e2e/framework/test_context.go

```diff
@@ -319,6 +322,8 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.StringVar(&TestContext.SnapshotControllerPodName, "snapshot-controller-pod-name", "", "The pod name to use for identifying the snapshot controller in the kube-system namespace.")
 	flags.IntVar(&TestContext.SnapshotControllerHTTPPort, "snapshot-controller-http-port", 0, "The port to use for snapshot controller HTTP communication.")
+
+	flags.BoolVar(&TestContext.EnableGcePdDriver, "enable-gcepd-driver", false, "Enable the gcepd volume driver. This should be used only on GCE clusters where the PD CSI driver is installed. See https://github.com/kubernetes/cloud-provider-gcp/tree/master/cluster/addons/pdcsi-driver for an example.")
```

Comment:
can you make this a generic arg that takes a comma separated string of driver names? I'm thinking we may want to move all the platform-specific drivers to be off by default in the future.

Reply:
Good idea.